### ISSUE TITLE
chromeinspector-20.2.0.jar forces slf4j binding

### ISSUE THREAD

**Issue by GenghisChen** [oracle/graal#2855]
Hello 
The chromeinspector-20.2.0.jar contains slf4j package structure and binding.
This means that a project containing chrome inspector, cannot use its own binding; getting this instead:
```
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:chromeinspector-20.2.0.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:log4j-slf4j-impl-2.13.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.SimpleLoggerFactory]
```

If slf4j is required for chrome inspector, it should be a dependency

Thanks
Chen

**Comment by chumer** [comment]
@entlicher is this dependency an optional one?

It seems the Maven descriptor does not show these dependencies. I think this should be fixed.
See:
https://repo1.maven.org/maven2/org/graalvm/tools/chromeinspector/20.2.0/chromeinspector-20.2.0.pom


FYI: If you use GraalVM you don't have any of these problems. There the inspector and JS classpath is fully separated from the application.

**Comment by entlicher** [comment]
The chromeinspector.jar was originally designed as self-contained. It bundles nano-httpd and also java-websocket, which requires the slf4j. We'd need to remove all the libraries from it.

**Comment by entlicher** [comment]
The maven descriptor does not show these dependencies, because there are no such dependencies when it contains the libraries already. We need to separate that.

**Comment by radeusgd** [comment]
> FYI: If you use GraalVM you don't have any of these problems. There the inspector and JS classpath is fully separated from the application.

Are you sure this is the case? Seems like it should be, but when I try to run a Truffle language that includes its own logger (temporarily, for testing purposes), I get the following output:
```
> java -jar -Dpolyglot.engine.IterativePartialEscape=true -Dpolyglot.engine.BackgroundCompilation=false -Dtruffle.class.path.append=runtime.jar runner.jar --repl
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/radeusgd/NBO/JVM/graalvm-ce-java11-20.2.0/tools/chromeinspector/chromeinspector.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/radeusgd/NBO/enso/runtime.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.SimpleLoggerFactory]
```

Am I doing something wrong? How can I isolate from the `chromeinspector` (as I'm not even using it)?

Just to be sure:
```
> java --version
openjdk 11.0.8 2020-07-14
OpenJDK Runtime Environment GraalVM CE 20.2.0 (build 11.0.8+10-jvmci-20.2-b03)
OpenJDK 64-Bit Server VM GraalVM CE 20.2.0 (build 11.0.8+10-jvmci-20.2-b03, mixed mode, sharing)
```

**Comment by marinier** [comment]
I'm running into another issue caused by this same problem. Specifically, I have a Java 8 library that depends on graal's `js` and `js-script-engine` (as opposed to using Nashorn) so that projects on newer JDKs (e.g., JDK 15) can still use it. While not strictly required, I was also depending on `chromeinspector` because it caused additional log printing to occur that was useful. The error I got in a Java 11 project was:

```
The package org.slf4j is accessible from more than one module: <unnamed>, org.slf4j
```

I think this is because the Java 8 library also depends on slf4j, and thus it was getting put in the `<unnamed>` module there, but `chromeinspector` is putting it in `org.slf4j` module?

The suggested fix above of not putting slf4j directly in chromeinspector seems correct to me -- slf4j should just be a dependency, which would allow it to be handled by maven and eclipse (which is what I'm using) correctly.

As a workaround for now, I've removed the `chromeinspector` dependency, which allows things to work.
